### DISCUSSION TITLE
[next/image] Allow delay before `placeholder="blur"` is removed (enable smooth image fades)

### DISCUSSION THREAD

**Discussion by lacymorrow** [vercel/next.js#32784] [Ideas]
### Describe the feature you'd like to request

I want to use `next/Image` to nicely transition between the placeholder image and the original as it becomes loaded.

Here is an example: https://codepen.io/darajava/pen/GRZzpbB?editors=0110

### Describe the solution you'd like

An easy fix would be to allow a delay before the background image is removed, so users may transition the original image in before the placeholder is remove abruptly.

### Describe alternatives you've considered

Using two regular `<img>` tags positioned above each other.

**Comment by mercteil** [comment]
Facing same issue with a "slide in" animation, where the animation is kind of interrupted half way by the rerendering of the image wrapper after blur. Which leads to a layout shift/jumping effect. It would be nice to delay the exchange of the blurred and the proper image by the time of executed animation.
